program/llvm-toolchain-ppl.md

```diff
@@ -131,18 +133,21 @@ A processor is only useful with a robust, up-to-date, proven compiler tool chain
 
 Furthermore, the licensing scheme of LLVM (Apache license) makes it particularly appealing to industrial players.  Because it does not require the disclosure of compiler source code, chip manufacturers and system integrators can better protect their differentiating Intellectual Property assets.  This licensing scheme also creates additional opportunities for software companies which can provide high added-value services in the area of compiler tuning for hardware/system-centric companies.
 
+Generated code for Embench tests by LLVM 11.0 is 4% quicker that GCC 10.2 code. This is an additional reason to offer LLVM support for CORE-V processors. 
+
 ## Industry landscape
 
-The "original" compiler used for the development of RISC-V cores is GCC.  However, LLVM offers a licensing scheme that is more appealing to the processor industry as it does not require the disclosure of critical vendor-specific intellectual property (IP) when distributing the software.  LLVM-based compiler chains appear to be at the foundation of many market leader offerings and it is expected that this will also be the case for RISC-V / CORE-V solutions.
+The "original" compiler used for the development of RISC-V cores is GCC licensed under the GNU General Public License, requiring vendors to make source code available.  LLVM offers a "permissive" licensing scheme which some customers find more attractive, since it does not require the source to be made available. As a result LLVM-based compiler chains appear to be at the basis (in whole or in part) of a number of proprietary compilers.  The downside is that there is less pressure on companies to contribute back to LLVM development.  This project will give OpenHW Group members a choice over which approach to adopt (make their toolchain code available or not).
 
 ### Related efforts to be described
 
-The upstream LLVM tool chain project already supports standard RISC-V, including all ratified ISA extensions.  RISC-V support in LLVM is strongly compliant to the RISC-V specification, but relatively immature in comparison to other architectures as x86, Arm and MIPS.
+The upstream LLVM tool chain project already supports standard RISC-V, including all ratified ISA extensions.  RISC-V support in LLVM is strongly compliant to the RISC-V specification, but relatively immature in comparison to older architectures as x86, Arm and MIPS.
+Moreover, LLVM follows closely new ISA extension discussions as it already provides support for not yet ratified extensions like Bit Manipulation and Vector ones.
 
-Currently LLVM does not support CORE-V specific features or extensions, and subsequently, its test suites do not include any CORE-V or PULP-specific tests.
+The support of CORE-V specific features or extensions by LLVM is in its infancy (e.g. hardware loop support just added in the assembler), and subsequently, its test suites do not include any CORE-V or PULP-specific tests.
 
```

Comment:
Actually they do have tests for hardware loop - no commits without associated tests!

Reply:
@ASintzoff Thanks for the quick turnaround. I'm happy for this to be committed. Added one small comment - when we added the Hardware Loop support, we added 15 CORE-V specific tests to the _lit_ regression tests.

@duncanbees - can you commit this.